common/config/data.tsx

```diff
@@ -5,6 +5,10 @@ import { GasPriceSetting } from 'types/network';
 import { makeExplorer } from 'utils/helpers';
 import NewTabLink from 'components/ui/NewTabLink';
 
+// tslint:disable-next-line:no-http-string
+export const IPFS_GATEWAY = 'http://127.0.0.1:8080';
```

Comment:
I think we might be defaulting to `https://ipfs.infura.io` to kick things off.

Reply:
Cool, should i change this now then? @skubakdj  @dternyak

Reply:
Yes, IMO. I believe we confirmed with @tayvano on using Infura in the MVP.

Reply:
Yup, I'm cool using Infura for MVP. Ideally we would set up our own stuff and use that but this is fine for the time being, especially until we have Mia and DS back 100%. 👍